apidocs/code-references.md

```diff
@@ -0,0 +1,6 @@
+<blockquote>
+    <h3><span>🚧</span>Use ld-find-code-refs</h3>
```

Comment:
I don't know if I'm sold on using these emoji in the headers. Seems too "cute" to me.

Reply:
I was just trying to port over the existing documentation which uses these, e.g. https://apidocs.launchdarkly.com/reference#rate-limiting

Reply:
I'm happy to address these emojis in a follow-up PR. Let's iterate.